build-support/ci.py

```diff
@@ -37,8 +37,8 @@ class PythonVersion(Enum):
   py36 = "3.6"
   py37 = "3.7"
 
-  def __str__(self):
-    return self.value
+  def __str__(self) -> str:
```

Comment:
Another potential bug caught? Nice!

Reply:
Yes! Didn't fix any actual bug in our code, but a potential one yes.

What's going on is that an `Enum`'s value is type `Any`. Above, we always are setting it to `str`, but you could just as well do something like `py36 = 3.6` (i.e. `float`). MyPy doesn't infer that we are always using `str`, because there is 0 guarantee that we will continue to do this. It is totally possible in Python to set one enum value as `str` and the next to `float`. So, this fix is a bit conservative but technically is correct to do.